core/keeper.go

```diff
@@ -72,24 +72,26 @@ func GetPrioritisedFTSOContract(blockTime *big.Int) string {
 	}
 }
 
-func GetMaximumMintRequest(blockNumber *big.Int) *big.Int {
-	switch {
-	default:
-		maxRequest, _ := new(big.Int).SetString("50000000000000000000000000", 10)
-		return maxRequest
+func GetMaximumMintRequest(blockNumber *big.Int, evm *vm.EVM) *big.Int {
+	maxRequest, _ := new(big.Int).SetString("50000000000000000000000000", 10)
+
+	if evm.ChainConfig().IsFlareHardFork1(evm.Context.Time) {
+		maxRequest, _ = new(big.Int).SetString("90000000000000000000000000", 10)
 	}
+
+	return maxRequest
 }
 
-func triggerKeeper(evm EVMCaller) (*big.Int, error) {
+func triggerKeeper(evmCaller EVMCaller) (*big.Int, error) {
```

Comment:
`EVMCaller` is an interface that represents the EVM already. There is no need to rename it, because the concrete underlying type is the EVM.